dev/encoding/vqgan-jax-encoding-yfcc100m-streaming.ipynb

```diff
@@ -0,0 +1,604 @@
+{
```

Comment:
<p>I didn't test how long it takes with modified version above for streaming (it is ignored since I have all data locally).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/borisdayma/dalle-mini/pull/84/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Reply:
<p>I can try that. I suppose the key will be to verify whether <code>num_proc</code>  is still useful, given that the generator is sequential. Perhaps the zip file contents are copied to the memory space of each process and everything works fast, even if memory usage increases. I'm still not familiar with Python multiprocessing, so it will be a fun test to make.</p>